labgrid/pytestplugin/reporter.py

```diff
@@ -3,6 +3,7 @@
 import os
 import re
 import colors
+import curses
```

Comment:
This new dependency needs to be described in setup.py and the requirements files as well. We should only import this if the `ColoredStepReporter`  is used and raise an appropriate error message if curses is not available, with a fallback to the old behaviour of assuming 256 colors.

Reply:
`curses` is part of the Python standard library.

Reply:
Nevertheless, it might be disabled when compiling Python, so I pushed a fix for that nevertheless.